ngraph/src/ngraph/pass/pass.hpp

```diff
@@ -32,8 +38,8 @@ namespace ngraph
         class PassBase;
         class ModulePass;
         class FunctionPass;
-        class NodePass;
-        class CallGraphPass;
+        class NodePass NGRAPH_DEPRECATED("Use MatcherPass or FunctionPass instead.");
+        class CallGraphPass NGRAPH_DEPRECATED("Use MatcherPass or FunctionPass instead.");
```

Comment:
@ilyachur @GlebKazantaev when it's going to be removed? only in 2022.1? Or we can remove it's usage and remove even in this release?

Reply:
I hope that we can remove it in this release. I know that some plugins has NodePass transformations. So it will be removed in a separate request.